spikeLog.txt

```diff
@@ -0,0 +1,52 @@
+Fri Oct 01 2021 12:08:05 GMT-0500 (Central Daylight Time) ~~ {"355759615268356097":{"name":"brandon.ingli","wallet":392},"255826944262799360":{"name":"maxwell","wallet":45},"579523103772901386":{"name":"Trick","wallet":50},"441023513382354944":{"name":"Becca","wallet":62},"204792522655006720":{"name":"ShaneCBA","wallet":161},"616419680365838347":{"name":"jpham35","wallet":10},"302664718038859781":{"name":"Eddιe♡","wallet":19249},"163124090159366144":{"name":"Awesopossum","wallet":27},"456851693972488192":{"name":"Mistaflowas","wallet":46},"371854362575896596":{"name":"toons","wallet":5},"105862054237208576":{"name":"Ethereal","wallet":2},"450052140124078093":{"name":"notavirus","wallet":54},"755612893046636585":{"name":"new~weiv","wallet":7},"139604789829959681":{"name":"Killexcl","wallet":7},"484572437716140052":{"name":"ilharper","wallet":24},"156112651481513985":{"name":"Moo_Cactus","wallet":13},"undefined":{"name":"undefined","wallet":-25103},"89925872500346880":{"name":"Alex Sweeney","wallet":13},"285373323418796042":{"name":"gammaking","wallet":18},"146375582920212480":{"name":"LunaBee","wallet":5},"370217540800086016":{"name":"Azeeza E.","wallet":1},"372005307116945408":{"name":"the wrong sOÜÜÜP","wallet":16},"337616280519573506":{"name":"MHM","wallet":1},"694727575041212476":{"name":"tylerrbeauregard","wallet":2}} > ../records.json
```

Comment:
This file needs to be ignored.

- [ ]  Add `spikeLog.txt` to `.gitignore`
- [ ] `git rm --cached spikeLog.txt` to remove it from tracking